docs/content/setup/_index.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Setup"
+date = 2020-05-26T20:55:04+03:00
+weight = 5
+chapter = true
+pre = '<i class="fas fa-cogs"></i> '
+tags = ["setup"] 
++++
+
+# Setting up Infection Monkey
+
+Setting up Infection Monkey is really easy! First, you need to {{% button href="https://infectionmonkey.com/" icon="fas fa-download" %}}download the Infection Monkey from our site{{% /button %}}.
```

Comment:
Why not https://www.guardicore.com/infectionmonkey/#download?

Reply:
Done